src/apps/core/admin.py

```diff
@@ -57,19 +71,53 @@ class DataCatalogAdmin(AbstractDatasetPropertyBaseAdmin, SimpleHistoryAdmin):
         "created",
         "modified",
     )
-    filter_horizontal = ("language",)
+    autocomplete_fields = ("language",)
 
 
 @admin.register(MetadataProvider)
 class MetadataProviderAdmin(admin.ModelAdmin):
     list_display = ("user", "organization")
+    list_filter = ("created", "modified")
+    search_fields = ("organization", "user__username", "user__email")
+
+
+class CoreRefDataAdminMixin:
+    search_fields = ["pref_label__values"]
```

Comment:
add url to search_fields
```suggestion
    search_fields = ["pref_label__values",  "url"]
```

Reply:
Done in latest commit